cumulus/pallets/parachain-system/Cargo.toml

```diff
@@ -125,3 +125,5 @@ try-runtime = [
 	"polkadot-runtime-parachains/try-runtime",
 	"sp-runtime/try-runtime",
 ]
+
+transaction-index = []
```

Comment:
Need to be enabled in the omni node.

Also please some docs on what this does would be nice.